suite-native/module-accounts-import/src/accountsImportThunks.ts

```diff
@@ -59,7 +59,7 @@ export const importAccountThunk = createThunk(
                     deviceState,
                     discoveryItem: {
                         index: deviceNetworkAccounts.length, // indexed from 0
-                        path: accountInfo?.path ?? '',
+                        path: (accountInfo?.path ?? '') as Bip43Path,
```

Comment:
unclear if `''` can happen, and how the `path` is used, but hey, the code was already like that; let's not fix what is probably not broken :shrug: